portal/templates/portal/mouseflow.html

```diff
@@ -0,0 +1,9 @@
+<script type="text/javascript">
+  var _mfq = _mfq || [];
+  (function() {
+    var mf = document.createElement("script");
+    mf.type = "text/javascript"; mf.async = true;
+    mf.src = "//cdn.mouseflow.com/projects/4e6407b5-890c-4dfc-9b50-fc14eeeeb2eb.js";
```

Comment:
Absolutely. Wasn't a security concern, just if someone else runs their own
instance of the portal might they be confused as to what's happening to
their heat maps? And will we be confused by another (potentially different
version) appearing in our reports?

On Friday, 29 April 2016, Mike Bryant <notifications@github.com> wrote:

> In portal/templates/portal/mouseflow.html
> <https://github.com/ocadotechnology/codeforlife-portal/pull/175#discussion_r61606727>
> :
>
> > @@ -0,0 +1,9 @@
> > +<script type="text/javascript">
> > +  var _mfq = _mfq || [];
> > +  (function() {
> > +    var mf = document.createElement("script");
> > +    mf.type = "text/javascript"; mf.async = true;
> > +    mf.src = "//cdn.mouseflow.com/projects/4e6407b5-890c-4dfc-9b50-fc14eeeeb2eb.js";
>
> It's client-side anyway, so everyone will be able to see this token, and I
> guess send fake data to mouseflow - like any client-side analytics
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ocadotechnology/codeforlife-portal/pull/175/files/c169f899b052d26af489875f623bdc8e5004ac98#r61606727>
>